integration/presto/src/main/prestodb/org/apache/carbondata/presto/readers/SliceStreamReader.java

```diff
@@ -124,6 +124,17 @@ public void putByteArray(int rowId, int count, byte[] value) {
     }
   }
 
+  @Override
+  public void putAllByteArray(byte[] data, int offset, int length) {
```

Comment:
Need to add this method  in prestosql SliceStreamReader.java as well?

Reply:
done